plugin.video.infowars/default.py

```diff
@@ -354,65 +364,71 @@ def Menu_MainMenu(): #The Main Menu
     li_url = IW_addon.build_plugin_url(queries)
     #xbmcplugin.addDirectoryItem(int(sys.argv[1]), li_url, listitem,isFolder=True)
     #IW_addon.add_directory({'mode': 'NightlyNewsSubMenu','title':'Infowars Nightly News'},{'title':  cFL_('Infowars Nightly News','red')},is_folder=True,img=IW_artIcon,fanart=IW_artFanart)
-    IW_addon.add_directory({'mode': 'ClipsSubMenu','title':'Infowars Clips'},{'title':  cFL_('Infowars Clips','yellow')},is_folder=True,img=IW_artIcon,fanart=IW_artFanart)
+    #IW_addon.add_directory({'mode': 'ClipsSubMenu','title':'Infowars Clips'},{'title':  cFL_('Infowars Clips','yellow')},is_folder=True,img=IW_artIcon,fanart=IW_artFanart)
     IW_addon.add_directory({'mode': 'DocSubMenu','title':'Acclaimed Documentaries'},{'title':  cFL_('Acclaimed Documentaries','blanchedalmond')},is_folder=True,img=IW_artIcon,fanart=IW_artFanart)
-    IW_addon.add_directory({'mode': 'HistoricShowsSubMenu','title':'Past Alex Jones Shows(video)'},{'title':  cFL_('Past Alex Jones Shows (Video)','yellow')},is_folder=True,img=AJSIcon,fanart=IW_artFanart)
-    IW_addon.add_directory({'mode': 'HistoricShowsAudioSubMenu','title':'Past Alex Jones Shows(video)'},{'title':  cFL_('Past Alex Jones (Audio)','yellow')},is_folder=True,img=AJSIcon,fanart=IW_artFanart)
+    #IW_addon.add_directory({'mode': 'HistoricShowsSubMenu','title':'Past Alex Jones Shows(video)'},{'title':  cFL_('Past Alex Jones Shows (Video)','yellow')},is_folder=True,img=AJSIcon,fanart=IW_artFanart)
+    #IW_addon.add_directory({'mode': 'HistoricShowsAudioSubMenu','title':'Past Alex Jones Shows(video)'},{'title':  cFL_('Past Alex Jones (Audio)','yellow')},is_folder=True,img=AJSIcon,fanart=IW_artFanart)
     
     eod()
+""" 
+#####################################################################################
+########### OLD CODE LEFT IN CASE IW IS UNBANNED BY THE INTERNET SOMEDAY ############
+#####################################################################################
 
 def Info_Wars_Live_Sub_Menu(title=''): #The Main Menu
     WhereAmI('@ Info Wars Live')
-    
+    http = urllib3.PoolManager()
     url = 'https://www.infowars.com/watch-alex-jones-show/'
-    response = urllib2.urlopen(url)
-    if response and response.getcode() == 200:
-        content = response.read()
+    response = http.request('GET', url)
+    if response:
+        content = response.data
         videos= find_multiple_matches(content,"<ul class=\"show__footer\"(.*?)</ul>")
         sources = []
         for temp, entry in enumerate(videos): 
             if temp == 0:
                 video_id = find_single_match(entry,"data-youtube-src=\"https://www.youtube.com/embed/(.*?)\"")
                 url = "plugin://plugin.video.youtube/play/?video_id=%s" % video_id
                 #IW_addon.log('SPINAL VIDEO_ID IS %s' % video_id)
-                PlayURL(url)
+                xbmc.Player().play(url)

```

Comment:
please don't use xbmc.Player().play() from plugins. This will create a new entry in the database each time a file is played. In plugins you should `setProperty("isPlayable", "true")` in the listitem and route the item to `xbmcplugin.setResolvedUrl()`. This way you request the playback of a `plugin://` path which is unique.

Reply:
Previous code can be looked up in the version code system you use for developing the add-on.

Reply:
Ya for sure, I was being lazy.  Removing now.